spark/src/test/scala/org/apache/spark/sql/CometTestBase.scala

```diff
@@ -249,7 +249,7 @@ abstract class CometTestBase
     var dfSpark: Dataset[Row] = null
     withSQLConf(
       CometConf.COMET_ENABLED.key -> "false",
-      "spark.sql.extendedExplainProvider" -> "") {
+      "spark.sql.extendedExplainProviders" -> "") {
```

Comment:
I agree, and do we really need to pass this param with blank value?

Reply:
Thanks,

> nit: could we make this literal as a constant 

I added the constant in a shim because it is defined in Spark 4.0

> do we really need to pass this param with blank value?

Yes, the caller of this method (`CometExpressionSuite`) defines a value and need to overwrite.